src/main/java/com/github/jacksonhoggard/voodoo2d/engine/GameEngine.java

```diff
@@ -44,34 +40,24 @@ public void run() {
 
     protected void init() throws Exception {
         window.init();
-        timer.init();
         mouseInput.init(window);
         gameLogic.init(window);
-        lastFps = Timer.getTime();
         fps = 0;
     }
 
     protected void gameLoop() {
-        float elapsedTime;
-        float accumulator = 0f;
-        float interval = 1f / TARGET_UPS;
-
-        boolean running = true;
-        while (running && !window.windowShouldClose()) {
-            elapsedTime = timer.getElapsedTime();
-            accumulator += elapsedTime;
+        double time = 0;
+        while (!window.windowShouldClose()) {
 
             input();
-
-            while (accumulator >= interval) {
-                update(interval);
-                accumulator -= interval;
-            }
-
+            update();
             render();
 
-            if (!window.isvSync()) {
-                sync();
+            // Fixed update gets called 60 times a second
+            time += Timer.getDeltaTime();
+            if (time > 0.016) {
+                fixedUpdate();
+                time = 0;
```

Comment:
fixedUpdate should be called before update. This seems to fix the stuttering animations and brings the frame rate up to 60.